draft-ietf-webtrans-overview.md

```diff
@@ -260,7 +300,67 @@ Streams SHOULD be sufficiently lightweight that they can be used as messages.
 Data sent on a stream is flow controlled by the transport protocol.  In addition
 to flow controlling stream data, the creation of new streams is flow controlled
 as well: an endpoint may only open a limited number of streams until the peer
-explicitly allows creating more streams.
+explicitly allows creating more streams.  From the perspective of the client,
+this is presented as a size-bounded queue of incoming streams.
+
+Any WebTransport protocol SHALL provide the following operations on the
+session:
+
+create a unidirectional stream
+: Creates an outgoing unidirectional stream; this operation may block until the
+  flow control allows for it to be completed.
+
+create a bidirectional stream
+: Creates an outgoing bidirectional stream; this operation may block until the
+  flow control allows for it to be completed.
+
+receive a unidirectional stream
+: Removes a stream from the queue of incoming unidirectional streams, if one is
+  available.
+
+receive a bidirectional stream
+: Removes a stream from the queue of incoming unidirectional streams, if one is
+  available.
```

Comment:
Does that return stream coint credit to the peer? In used to implementations that only return the credit once the stream completes.

Reply:
That's currently implementation defined.  Chromium currently uses the latter approach (limit on total max open streams), but I can imagine implementations setting different limits for unaccepted streams vs total streams.